app/scripts/views/settings/emails.js

```diff
@@ -68,6 +70,16 @@ define(function (require, exports, module) {
     _isSecondaryEmailEnabled () {
       // Only show secondary email panel if the user is in a verified session and feature is enabled.
       const account = this.getSignedInAccount();
+
+      // This matches the enable feature flag for secondary emails, on the auth-server
+      // https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L762
+      // By adding the check here, we do not have to issue a request to the auth-server and risk
+      // getting a 502/502 response. While this solution works, a longer term fix
+      // should be in place for handling feature rollouts.
+      if (! EMAIL_ENABLED_REGEX.test(account.get('email'))) {
+        return this.remove();
+      }
```

Comment:
Even though we're patching a problem with the release, I reckon we should still have some form of test coverage for this condition (assuming that it's straightforward to do so).